library/ssl_tls.c

```diff
@@ -3146,11 +3146,19 @@ int mbedtls_ssl_write_handshake_msg( mbedtls_ssl_context *ssl )
     /*
      * Sanity checks
      */
-    if( ssl->out_msgtype != MBEDTLS_SSL_MSG_HANDSHAKE &&
+    if( ssl->out_msgtype != MBEDTLS_SSL_MSG_HANDSHAKE          &&
```

Comment:
Why such strange formatting?

Reply:
Just taste, I usually prefer it to be aligned with the bracket in the next line.

Reply:
It's very inconsistent here:
3149 - aligning to next bracket
3156 - aligning to last &&
3165 - no alignment.
But that's just a cosmetic change, so that's not a blocker for me.